cmd/migrate/clickhouse.go

```
@@ -77,18 +77,30 @@ func runClickHouseMigrations(ctx context.Context, cfg *config.Configuration, dir
+	// once the pool is already shut down -- the process then deadlocks with
+	// "all goroutines are asleep" before a single migration file is applied.
+	// That is not a timeout: nothing recovers, and the Job burns its full
+	// deadline producing no output past the first log line.
+	if err := func() error {
+		bootstrap := *opts
+		bootstrap.Auth.Database = "default"
+		conn, err := clickhouse_go.Open(&bootstrap)
+		if err != nil {
+			return fmt.Errorf("open clickhouse: %w", err)
+		}
+		defer conn.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `conn.Close` is not checked

(errcheck)